### PR TITLE
Site improvements

### PR DESCRIPTION
**Pull request by doomspork**
RFC @pjhampton, @pragmaticivan 

This PR includes a couple of general site improvements:

Features:
- Added sitemap.xml
- Added robots.txt
- Collapse Advanced and Specific sections by default

Fixes:
- Fixed `hex.pm` link across multiple languages
- Fixed `control-structures.md` link across multiple languages

Please do not merge this yet, looking for feedback at the moment :grinning:

### CODE REVIEW

File: _assets/javascripts/app.js [hunk 1]
```diff
@@ -20,4 +20,7 @@ var toggle_section = function() {
 
 $(function() {
   $('.sidebar-nav-header').click(toggle_section);
+
+  $('.advanced .sidebar-nav-header').click();
```
Comment (doomspork) [inline _assets/javascripts/app.js]: This isn't pretty but it does the job.

Comment (pragmaticivan) [inline _assets/javascripts/app.js]: I was about to comment asking if bootstrap has a better way. But it works :)

File: _includes/sidebar.html [hunk 2]
```diff
@@ -43,5 +43,6 @@
     </p>
 
     <p>&copy; {{ site.time | date: '%Y' }} <a href="https://github.com/doomspork">Sean Callan</a></p>
+    <p><a href="/sitemap.xml">sitemap</a></p>
```
Comment (pragmaticivan) [inline _includes/sidebar.html]: Once It's a XML instead an html, is that required? My previous projects they used to have a html and xml version of sitemap.

Comment (doomspork) [inline _includes/sidebar.html]: This link probably isn't necessary now that the robots.txt links to the sitemap.

I'll :scissors:  it

File: _config.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ description:
   cn: Lessons about the Elixir programming language
   es: Lecciones acerca del lenguaje de programación Elixir
   pt: Lições sobre a linguagem de programação Elixir
-  vi: Lessons about the Elixir programming language
+  vi: Giáo trình cho ngôn ngữ lập trình Elixir
```
Comment (pjhampton) [inline _config.yml]: You know Vietnamese? :stuck_out_tongue_winking_eye:

**Comment by pjhampton** [comment]
LGTM! :+1:

File: _config.yml (hunk 3 above)
Comment (doomspork) [inline _config.yml]: Hah!  No, @joneslee85 pasted it into #152 :stuck_out_tongue:

File: _assets/javascripts/app.js (hunk 1 above)
Comment (doomspork) [inline _assets/javascripts/app.js]: I'm going to merge this as-is, we can always improve it later.

**Pull request merged**